include/NovelRT.Interop/Ecs/Audio/NrtAudioSystem.h

```diff
@@ -0,0 +1,32 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#ifndef NOVELRT_INTEROP_ECS_AUDIO_AUDIOSYSTEM_H
+#define NOVELRT_INTEROP_ECS_AUDIO_AUDIOSYSTEM_H
+
+#include "../../NrtTypedefs.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    NrtAudioSystemHandle Nrt_AudioSystem_Create();
```

Comment:
I might be just being blind here, but is there an appropriate destroy/delete function for this?

Reply:
Noooooooo :^) added it in the next commits